### PR TITLE
backendSize for workspaces

### PR DESCRIPTION
**Pull request by ErikZigo**
Jira: https://keboola.atlassian.net/browse/KBC-1589

Before asking for review make sure that:

- [x] ~New client method(s) has tests~
- [x] Apiary file is updated
- [x] You declared if there is a BC break or not (will affect next release of a client)

---

### CODE REVIEW

File: tests/Backend/Workspaces/SnowflakeDynamicBackendsTest.php [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+            'backend' => StorageApiTestCase::BACKEND_SNOWFLAKE,
+            'backendSize' => $backendSize,
+        ]);
+
+        $this->assertSame('snowflake', $workspace['connection']['backend']);
+        $this->assertSame($expectedBackendSize, $workspace['backendSize']);
+        $this->assertStringEndsWith($expectedWarehouseSuffix, $workspace['connection']['warehouse']);
+    }
+
+    public function testWorkspaceLoadDataFromProjectWithDynamicBackends()
```
[the remark this answers is not in the document]
Comment (ErikZigo) [reply tests/Backend/Workspaces/SnowflakeDynamicBackendsTest.php]: Pojmenoval jsem to nakonec trochu jinak. Snad uz je to zrejmejsi.

File: tests/Backend/Workspaces/SnowflakeDynamicBackendsTest.php [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+                        'destination' => self::WORKSPACE_TABLE_NAME
+                    ],
+                ],
+            ]
+        );
+
+        $this->assertDataInWorkspace($backend);
+    }
+
+    public function testWorkspaceLoadSharedDataFromProjectWithoutDynamicBackends()
```
[the remark this answers is not in the document]
Comment (ErikZigo) [reply tests/Backend/Workspaces/SnowflakeDynamicBackendsTest.php]: Pojmenoval jsem to nakonec trochu jinak. Snad uz je to zrejmejsi.

File: tests/Backend/Workspaces/SnowflakeDynamicBackendsTest.php [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+        $this->_client->createTable(
+            $bucketId,
+            'languages',
+            new CsvFile(self::SOURCE_DATA_FILE_PATH)
+        );
+
+        $this->_client->shareBucket(
+            $bucketId,
+            [
+                'sharing' => 'organization-project',
```
Comment (ErikZigo) [inline tests/Backend/Workspaces/SnowflakeDynamicBackendsTest.php:95]: Opraveno nastaveni typu sdileni

File: tests/Backend/Workspaces/SnowflakeDynamicBackendsTest.php [hunk 4]
```diff
@@ -0,0 +1,345 @@
...
+        $this->client2->createTable(
+            $bucketId,
+            'languages',
+            new CsvFile(self::SOURCE_DATA_FILE_PATH)
+        );
+
+        $this->client2->shareBucket(
+            $bucketId,
+            [
+                'sharing' => 'organization-project',
```
Comment (ErikZigo) [inline tests/Backend/Workspaces/SnowflakeDynamicBackendsTest.php:161]: Opraveno nastaveni typu sdileni

**Review by fejjo — APPROVED** [review]
lgtm

**Pull request merged**